include/netutils/webclient.h

```diff
@@ -191,13 +191,17 @@ struct webclient_context
    *   url          - A pointer to a string containing the full URL.
    *                  (e.g., http://www.nutt.org/index.html, or
    *                   http://192.168.23.1:80/index.html)
+   *   unix_socket  - If not NULL, the path to an AF_LOCAL socket.
    *   headers      - An array of pointers to the extra headers.
    *   nheaders     - The number of elements in the "headers" array.
    *   bodylen      - The size of the request body.
    */
 
   FAR const char *method;
   FAR const char *url;
+#if defined(CONFIG_WEBCLIENT_NET_LOCAL)
+  FAR const char *unix_socket;
```

Comment:
who will set unix_socket? I don't find the related code.

Reply:
the calling app.